Gemfile.lock

```diff
@@ -1,45 +0,0 @@
-GEM
```

Comment:
@bastelfreak then the solution is to specify the supported ruby version :) this stuff is made to work with AIO, that means one thing